main.tf

```diff
@@ -1,42 +1,22 @@
 terraform {
-  required_version = ">= 0.14.11"
+  required_version = ">= 0.12.17"
   required_providers {
     aws = ">= 3.0"
   }
 }
 
-locals {
-  filename = "${path.module}/lambda/function.zip"
+module "acs" {
+  source            = "github.com/byu-oit/terraform-aws-acs-info?ref=v3.5.0"
+  vpc_vpn_to_campus = true
 }
 
-resource "aws_lambda_function" "logging" {
-  function_name    = "${var.app_name}-logs-to-humio"
-  filename         = local.filename
-  handler          = "index.handler"
-  runtime          = "nodejs14.x"
-  role             = aws_iam_role.logging.arn
-  source_code_hash = filebase64sha256(local.filename)
-  timeout          = var.timeout
-  memory_size      = var.memory_size
-  tags             = var.tags
-
-  environment {
-    variables = {
-      ENV                = var.app_env
-      HUMIO_INGEST_TOKEN = var.humio_ingest_token
-      SUB_IDX_NM         = var.sub_idx_nm
-    }
-  }
-
-  vpc_config {
-    security_group_ids = [aws_security_group.logging.id]
-    subnet_ids         = var.private_vpn_subnet_ids
-  }
+data "local_file" "cloudformation" {
```

Comment:
is using cloudformation the recommended way to do all this? or in other words; did humio give us the cloudformation template?

Reply:
Yes, this is their solution that we're wrapping with terraform

Reply:
OK, it makes my skin crawl, but if that's their solution then "so be it" 😅